README.md

```diff
@@ -175,7 +175,7 @@ var storage = multer.diskStorage({
     cb(null, '/tmp/my-uploads')
   },
   filename: function (req, file, cb) {
-    cb(null, file.fieldname + '-' + Date.now())
+    cb(null, file.fieldname + '-' + Date.now() + '-' + Math.random())
```

Comment:
Hmm, this will produce filenames like: `foo-1581931472193-0.5359727197825674`. I'm not sure how that is being handled since essentially the file extension of the file is now `.5359727197825674` 🤔